lib/components/narrative/default/default-itinerary.js

```diff
@@ -48,18 +90,20 @@ const ITINERARY_ATTRIBUTES = [
         else return formatTime(itinerary.startTime, options)
       }
       return (
-        <span>
-          {formatTime(itinerary.startTime, options)}
-          —
-          {formatTime(itinerary.endTime, options)}
-        </span>
+        formatTime(itinerary.startTime, itinerary.endTime, options.timeFormat)
       )
     }
   },
   {
     id: 'cost',
     order: 2,
-    render: (itinerary, options) => getTotalFareAsString(itinerary)
+    render: (itinerary, options) => {
+      // Get unformatted transit fare portion only (in cents).
+      const { transitFare } = calculateFares(itinerary)
+      return (
+        <FormattedNumber value={transitFare / 100} style='currency' currencyDisplay='narrowSymbol' currency={options.currency} />
```

Comment:
Line too long.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2